dev/refresh_source.py

```diff
@@ -434,7 +434,8 @@ def write_deck_run_script(self, repository):
 cd "$d"
 LOG_DIR=${{LOG_DIR:-../logs}}
 
-bash -c "(./start.sh >> '$LOG_DIR/{name}.log') 2>&1\
+npm install >& "$LOG_DIR/deck.log"
```

Comment:
I'm running from HEAD and have the right log files, deck starts up, and I see stuff at 9000 though I havent interacted with it (though I have before but not sure of the build purity). I'm not sure what was broken.

I guess start.sh is for those using the repository without refresh_source, so it is still relevant.
Would be nice for run_dev to wrap start.sh and just add the consistent logging.